test/external/sra-info/test-sra-info.cpp

```diff
@@ -495,6 +495,66 @@ FIXTURE_TEST_CASE(HasPhysicalQualities_Original, SraInfoFixture)
     info.SetAccession(Run_Multiplatform);
     REQUIRE( info.HasPhysicalQualities() );
 }
+#endif
+
+// Contents
+FIXTURE_TEST_CASE(Contents_Table, SraInfoFixture)
+{
+    info.SetAccession(Accession_Table);
+    SraInfo::Contents cnt = info.GetContents();
+    REQUIRE_EQ( Accession_Table, string(cnt -> name) );
+    REQUIRE_EQ( (int)kptTable, (int)(cnt -> dbtype) );
+    REQUIRE_EQ( 0, (int)(cnt -> fstype) );
+    REQUIRE_EQ( cca_HasChecksum_CRC | cca_HasLock | cca_HasMD5_File | cca_HasMetadata | cta_HasIndices,
+                (int)(cnt -> attributes) );
+    REQUIRE_NULL( cnt -> parent );
+    REQUIRE_NULL( cnt -> nextSibling );
+    REQUIRE_NULL( cnt -> prevSibling );
+
+    const KDBContents * child1 = cnt -> firstChild;
+    REQUIRE_NOT_NULL( child1 );
+    REQUIRE_EQ( string("ALTREAD"), string(child1 -> name) );
```

Comment:
I worry that the order of the children could change. There is nothing in the code that builds this tree that guarantees the order of the nodes. IIRC, KFS doesn't guarantee the order of directory visiting.